apps/avx-sales-api/src/users/entities/user.entity.ts

```diff
@@ -32,4 +34,35 @@ export class AVXUser implements IAVXUser {
       role: this.role,
     };
   }
+
+  static fromDbUser(user: User): AVXUser {
+    return new AVXUser(user);
+  }
+}
+
+@Entity({
+  name: 'users',
+  synchronize: false,
+})
+export class User extends BaseEntity implements IAVXUser {
+  @PrimaryGeneratedColumn()
+  _id: number;
+
+  @Column('uuid')
+  id: string;
+
+  @Column()
+  name: string;
```

Comment:
do we need to include {nullable: true} since it could be null? Not sure if planescale would throw error

Reply:
Yep, it will throw. Good point. This is just a basic scaffold for now.